src/components/tasks/card/index.tsx

```diff
@@ -72,6 +72,12 @@ const Card: FC<CardProps> = ({
 
     const [keyLongPressed] = useKeyLongPressed();
 
+    const [isEditMode, setIsEditMode] = useState(false);
+    const onEditRoute = () => {
+        setIsEditMode(true);
+    };
```

Comment:
- Move this below the `useEffect` ? As we generally have this hierarchy in a react component where we: 
```js
-- declare your state / hooks --
-- variables dependent on state / value from hooks --
-- event / action handlers --
-- jsx mark / return --